streams/readable-streams/floating-point-total-queue-size.js

```diff
@@ -0,0 +1,84 @@
+'use strict';
+
```

Comment:
How about a comment explaining why this stuff matters? Something like:

```javascript
// Due to the limitations of floating-point precision, the calculation of desiredSize sometimes gives different answers
// than adding up the items in the queue would. It is important that implementations give the same result in these edge
// cases so that developers do not come to depend on non-standard behaviour. See
// https://github.com/whatwg/streams/issues/582 and linked issues for further discussion.
```

It's unfortunate that we didn't keep all the discussion in one place, but we can't put that genie back in the bottle now.